Introduction.Rmd

```diff
@@ -137,64 +159,99 @@ Make sure you have a recent version of RStudio. You can check you have the right
+Additional Tasks".
+
+- Do _not_ select the box for "Edit the system PATH". devtools and RStudio should put Rtools on the `PATH` automatically when it is needed.
+- Do select the box for "Save version information to registry" (it should be selected by default).
+
+### macOS
+
+You need to install the Xcode Command Line Tools, which may already be installed. You can check if they are by running
+
+```{r, eval = FALSE}
+# TODO: replace with whatever results from https://github.com/r-lib/devtools/issues/1970
+devtools::has_devel()
+```
+
+If they are not installed you have a few options.
+
+- Minimalist approach (recommended): Install Xcode Command Line Tools.
+  In the shell:
+  ```shell
+  xcode-select --install
+  ```
+- Install the current release of full [Xcode from the Mac App Store](https://itunes.apple.com/ca/app/xcode/id497799835?mt=12). This includes a very great deal that you do not need, but it offers the advantage of App Store convenience.
+
+### Linux
+
+Make sure you've installed not only R, but also the R development tools. For example, on Ubuntu (and Debian) you need to install the `r-base-dev` package.
+
+### Verify system prep
 
 You can check that you have everything installed and working by running the following code:
 
 ```{r, eval = FALSE}
+# TODO: replace with whatever results from https://github.com/r-lib/devtools/issues/1970
 library(devtools)
 has_devel()
-#> '/Library/Frameworks/R.framework/Resources/bin/R' --vanilla CMD SHLIB foo.c 
-#> 
-#> clang -I/Library/Frameworks/R.framework/Resources/include -DNDEBUG 
-#>   -I/usr/local/include -I/usr/local/include/freetype2 -I/opt/X11/include
-#>   -fPIC  -Wall -mtune=core2 -g -O2  -c foo.c -o foo.o
-#> clang -dynamiclib -Wl,-headerpad_max_install_names -undefined dynamic_lookup
-#>   -single_module -multiply_defined suppress -L/usr/local/lib -o foo.so foo.o 
-#>   -F/Library/Frameworks/R.framework/.. -framework R -Wl,-framework 
-#>   -Wl,CoreFoundation
-[1] TRUE
+#> [1] TRUE
 ```
 
-This will print out some code which I use to help diagnose problems. If everything is ok, it will return `TRUE`. Otherwise, it will throw an error and you'll need to investigate what the problem is.
+If everything is ok, it returns `TRUE`. Otherwise, it will reveal some diagnostic info about the problem.
+
+### Startup configuration
 
-Since devtools is a workflow package, you may consider making it available automatically in all of your interactive R sessions. In general, it's not a great idea to load packages this way, as it invites you to create R scripts that don't reflect all of their dependencies via explicit calls to `library(foo)`. But devtools smooths the process of package development and is, therefore, unlikely to get baked into any analysis scripts. The pros may outweigh the cons for workflow packages you use often.
+You can attach devtools like so:
 
-You can attach devtools in all of your interactive session by putting these lines `.Rprofile`:
+```{r, eval = FALSE}
+library(devtools)
```

Comment:
Do we need `library(usethis)` too?

Reply:
No. Currently devtools re-exports all of usethis. The mechanics of that may change but presumably not the spirit.

This issue is the main point of this PR, so let's make sure that we are all on the same page about it.

Reply:
Surely it can only re-export the functions it knows about?

Reply:
It doesn't _technically_ re-export it, it attaches it

Reply:
And probably in the next devtools release I will just remove the old deprecated `use_*()` functions entirely from devtools and just put usethis in `Depends`.

Reply:
> It doesn't technically re-export it, devtools attaches usethis

Yes, right. This was poor wording on my part, hard-waving at the possibility that the mechanism by which devtools attaches usethis may change.